.npmignore

```diff
@@ -1,5 +1,8 @@
 node_modules
 coverage
+.nyc_output
 docs
 tests
 bench
+nunjucks
+scripts
```

Comment:
Worth noting, that instead of maintaining `.npmignore` it is easier to specify [`files`](https://github.com/mozilla/nunjucks/pull/1053#discussion_r162205497) property in `package.json`